src/Symfony/Component/Mailer/Event/MessageEvent.php

```diff
@@ -20,7 +20,7 @@
  *
  * @author Fabien Potencier <fabien@symfony.com>
  */
-class MessageEvent extends Event
+final class MessageEvent extends Event
```

Comment:
@fabpot there is a `MessageEvents` class in the same mailer namespace. This can easily be confused with the event class and should maybe be renamed or relocated or even made internal.

Reply:
isn't MessageEvents the class holding all event names as constants ? This is not internal

Reply:
The Mailer MessageEvents is a class used for logging and data collecting. It doesn't have anything to do with an event.